contents/customers/contra.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How Contra used session recordings to improve registrations by 30%
+customer: Contra
+logo: ../images/customers/contra/contra_logo.png
+featuredImage: ../images/customers/contra/contra_featured.png
+industries:
+    - Saas
+users:
+    - Product
+    - Engineering
+toolsUsed:
+    - Feature Flags
+    - Session Recording
+    - Product Analytics
+---
+
+Contra is a freelance marketplace for service independent creatives and engineers. It empowers its community to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com), has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.
```

Comment:
Remove word "service" - we don't use this term

Reply:
```suggestion
Contra is a freelance marketplace for independent creatives and engineers. It empowers its community to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com), has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.
```